src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add `logger` (constructor) property to `RoundRobinTransport`
+ * Add `subaccount` parameter to `MandrillApiTransport` payload
```

Comment:
What is actually added is support for specifying a subaccount using the `X-MC-Subaccount` header. That's the user-visible change (adding the parameter in the payload is describing the internal implementation, not the new feature)